HW2/P4/P4.txt

```diff
@@ -0,0 +1,13 @@
+I used the start/join method along with a list of semaphores to finish this problem. I think the start/join might be sufficient for this problem too because one of my friends used only start/join without semaphores to get almost the same speed up. I used start/join method because it is the most convenient way to solve this problem. 
```

Comment:
I ended up using Events instead of Semaphores. I thought it was a bit easier since we only needed one Event per thread/iteration. This looks great though!